### ISSUE TITLE
Advice on the cleanest way to use epel for centOS images

### ISSUE THREAD

**Issue by vbusch** [cekit/cekit#95]
Before using epel repo in centOS, you need to install the epel-release.  This requires 2 separate yum commands.  Making one command to install all the packages will fail.
```
yum -y install epel-release
yum -y install nodejs cyrus-sasl-lib npm
```
For now, I have created a base image that includes the epel-release.  This let me do it with packages.  And then other image.yaml files build from that image.

It would be nice, if it were possible to have the packages section, issue the 2 yum commands.  This would require an enhancement

It was also mentioned that this could be done in the modules section.  I am assuming that would entail having an install.sh script that issues the 2nd yum command?

**Comment by goldmann** [comment]
I talked about this today with @dbecvarik. The plan is to sit down, think about possible solutions and propose the best one. This can be solved in multiple ways, but in that meeting we couldn't find the best solution for this, so we postponed it to do a proper investigation.

**Comment by rcernich** [comment]
Would it be easy enough to try a custom template that separates the package installs, e.g. one ``yum install`` per package?  I'm not saying we take this approach, but it would be easy enough to test, without having to modify any concreate code (unless we can't specify a custom template).

**Comment by dbecvarik** [comment]
for a hacky solution, you can create a module which will do rpm -ivh $url_of_epel_repo, which will work for you if you need a quick solution. We will work on much suitable solution probably in few days.

**Comment by dbecvarik** [comment]
This should be solved in Cekit 2.0.
you can now request epel repository to be added via something like:

``` yaml
packages:
      repositories:
          - name: epel
          - rpm: epel-release-latest-7
```
